packages/cli/src/commands/generate/scaffold/scaffold.js

```diff
@@ -142,11 +143,13 @@ export const files = async ({
   docs,
   model: name,
   path: scaffoldPath = '',
-  tests = true,
+  tests,
   typescript = false,
   tailwind = false,
   force = false,
   nestScaffoldByModel,
+  stories,
+  serviceTests = true,
```

Comment:
this preserves the current operation mode, but because the DEFAULT_CONFIG  (packages/project-config/src/config.ts) sets `tests = true` the fixtures of the test-project change. We can either unify the `tests` flag, keep `tests` for the service and use `componentTests` for the component tests. I think unifying the flag makes the most sense, but it might be a breaking change.  @Tobbe Please advice,